dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -369,6 +369,93 @@ def diff(self, other_profile: NumericStatsMixin, options: Dict = None) -> Dict:
                 "and '{}'".format(cls.__name__, other_profile.__class__.__name__)
             )
 
+        new_self_histogram = {"bin_counts": None, "bin_edges": None}
```

Comment:
I think this is fine, but one thing I would consider is the rest of the api functions in diff are purely padding the data with all the checks within.

To keep things similar, unless the histograms were used elsewhere, I'd consider putting all of this in calculate_psi

Reply:
nah that's good -- lemme see what I can do right quick and push up